src/style.css

```diff
@@ -0,0 +1,100 @@
+html {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  height: 300px;
+  flex-basis: 300px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-grow: 1;
+  flex-shrink: 0;

```

Comment:
by the way, you can combine flex-grow, flex-shrink and flex-basis into one 'flex' rule
https://developer.mozilla.org/en-US/docs/Web/CSS/flex